src/utils/products.ts

```diff
@@ -139,14 +145,14 @@ export const baseProducts: Product[] = [
 		imageUrls: ['products/watch.png'],
 		categories: ['Accessories'],
 		variations: [],
-		price: { amount: 150, currency: 'USD' }
+		price: { amount: 150, currency: '$' }
 	}
 ];
 
 // Generate 45 Products by Multiplying Base Array
 export const products: Product[] = Array.from({ length: 4 }).flatMap((_, index) =>
 	baseProducts.map((product) => ({
 		...product,
-		name: `${product.name} #${index}`
```

Comment:
I would remove this all-together, no duplication. Let's ask chen for enough products to show  second page instead

Reply:
We only have like 12 products in the hardcoded list, removing multiplication will show disabled Pagination :/

Reply:
Having duplication adds code that noone will want when actually using template. They add product and see it 5 times, think page is broken - not use it. We really must remove it.

If we hate the fact pagination doesnt look well, we ask Chen for more products.
Also good idea to bring it up if it looks bad when it's single page, because many e-shops will only sell a few products, having this exact design.

Reply:
1. makes sense
2. no issues with pagination, my point was if we have only 12 items, then showing pagination component makes no sense because there won't be more pages.
3. I wanted to keep the demo/multiplied items just for demo

should I remove it altogether?

Reply:
2. Well real developers will have this concenr too then, if they have less than 12 items. Best to have design that looks good even for 3 or 4 items

3. I actually want to avoid this.I clicked around and thought its borken becasue I kept seeing the same products. To make wow effect, we either need more real products, or keep just one page. Or have exactly 14 items, so we have second page with at least 2 items